### ISSUE TITLE
Can’t install milvus-lite==2.4.10 

### ISSUE THREAD

**Issue by Zhang-Xinchen** [bRAGAI/bRAG-langchain#7]
**Describe the bug**
ERROR: Could not find a version that satisfies the requirement milvus-lite==2.4.10 (from versions: none)
ERROR: No matching distribution found for milvus-lite==2.4.10

I use pip install -r requirements.txt in virtual by anconda
The python version is Python 3.11.7

**Comment by tahababou12** [comment]
Yep - a few people have been experiencing similar dependency installation issues. I will release a new update soon.

Which operating system are you using?

**Comment by Zhang-Xinchen** [comment]
I am using Windows 11 system, but this is being executed in an Anaconda virtual environment. conda version is conda 24.9.2

**Comment by tahababou12** [comment]
Similar issue was reported in #4 - closing this issue
